satori/resources/resource_taxonomy_classifier.go

```
@@ -186,7 +186,8 @@ func resourceToClassifier(d *schema.ResourceData) (*api.TaxonomyClassifier, erro
 		}
 		if _, ok := d.GetOk("custom_config.0.values"); ok {
 			var values api.TaxonomyClassifierValues
-			values.Values = getStringListProp("custom_config.0.values", d)
+			valuesConfig, _ := getStringListProp("custom_config.0.values", d)
```

Why do you ignore the error?

---

Not interested for now